analyzers/rspec/cs/S2326_c#.json

```diff
@@ -12,5 +12,10 @@
   "defaultSeverity": "Major",
   "ruleSpecification": "RSPEC-2326",
   "sqKey": "S2326",
-  "scope": "All"
+  "scope": "All",
+  "securityStandards": {
+    "CERT": [
+      "MSC12-CPP."
+    ]
```

Comment:
Isn't this wrong? To have a cpp standard on a C# project?

Reply:
CERT removed manually.
There might be some more of them going away after RULEAPI-665 is fixed in further releases. There should be no harm, as they aren't user facing according to slack message.